modules/tinymce/src/core/main/ts/keyboard/FakeInputEvents.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import Editor from '../api/Editor';
+import { clone } from '../events/EventUtils';
+
+interface SpecificsInput {
+  data?: null | string;
+}
+
+const fireFakeInputEvent = (editor: Editor, inputType: string, specifics: SpecificsInput = {}) => {
+  const target = editor.getBody();
+  const overrides = {
+    bubbles: true,
+    composed: true,
+    data: null,
+    isComposing: false,
+    detail: 0,
+    view: null,
+    target,
+    currentTarget: target,
+    eventPhase: Event.AT_TARGET,
+    originalTarget: target,
+    explicitOriginalTarget: target,
+    isTrusted: true,
```

Comment:
Is this correct? As this won't be a trusted event since it's mocked so if someone tries to do something with it that requires that it'll fail.

Reply:
I guess it depends on how we view that property but it would make it possible to check for simulated events vs real events. But all events are somewhat simulated since they are cloned and dispatched though a event delegator. So then we should switch this to false for all events.

Reply:
Point... I guess leave it as is then :+1: